app.ts

```diff
@@ -648,7 +652,7 @@ async function main() {
 
     function renderGoldenLayout(config, metadata, req, res) {
         staticHeaders(res);
-        contentPolicyHeader(res);
+        contentPolicyHeader();
```

Comment:
I would prefer we instead add a _res unused parameter to contentPolicyHeader() to avoid confusion